jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -184,14 +185,8 @@ def vue_do_aper_phot(self, *args, **kwargs):
                 raise ValueError('Missing or invalid background value')
             comp_no_bg = comp.data - bg
 
-            # TODO: Use photutils when it supports astropy regions.
-            if not isinstance(reg, RectanglePixelRegion):
-                aper_mask = reg.to_mask(mode='exact')
-            else:
-                # TODO: https://github.com/astropy/regions/issues/404 (moot if we use photutils?)
-                aper_mask = reg.to_mask(mode='subpixels', subpixels=32)
-            npix = np.sum(aper_mask) * u.pix
-            img = aper_mask.get_values(comp_no_bg, mask=None)
+            aperture = regions2aperture(reg)
+            npix = len(aperture.to_mask(method='center').get_values(comp_no_bg)) * u.pix

```

Comment:
This is not equivalent to the old `regions` calculation but it is the closest I can find. The old way to do things was wrong for rectangle but it did provide fractional npix, which seems to matter a lot for ellipse (see updated tests in this PR).